python/ray/autoscaler/azure/config.py

```diff
@@ -59,169 +57,60 @@ def _configure_resource_group(config):
     resource_client.resource_groups.create_or_update(
         resource_group_name=resource_group, parameters=params)
 
-    return config
-
-
-def _configure_msi_user(config):
-    msi_client = _get_client(ManagedServiceIdentityClient, config)
-    resource_client = _get_client(ResourceManagementClient, config)
-    auth_client = _get_client(AuthorizationManagementClient, config)
+    # load the template
+    template_path = os.path.join(
+        os.path.dirname(__file__), 'azure-config-template.json')
+    with open(template_path, 'r') as template_file_fd:
+        template = json.load(template_file_fd)
+
+    # choose a random subnet
+    random.seed(resource_group)
+    parameters = {"subnet": "10.{}.0.0/16".format(random.randint(0, 254))}
```

Comment:
may want to exclude 0 specifically from the range of options

Reply:
happy to do so, but why?

Reply:
i'm assuming the most likely conflict will be with 10.0.0.x, so if we exclude that altogether it might be better